doozerlib/cli/get_nightlies.py

```diff
@@ -1,97 +1,186 @@
+from typing import List, Dict, Optional, Set, Tuple
+import asyncio
 import click
+import hashlib
 import json
 import sys
 from urllib import request
-from doozerlib.cli import cli
-from doozerlib import constants, util, exectools
-from doozerlib.rhcos import get_primary_container_name
+from doozerlib.cli import cli, click_coroutine
+from doozerlib.model import Model
+from doozerlib import constants, util, exectools, logutil
+from doozerlib.rhcos import RHCOSBuildInspector
 
+logger = logutil.getLogger(__name__)
 
-@cli.command("get-nightlies", short_help="Get sets of Accepted nightlies. A set contains nightly for each arch, "
-                                         "determined by closest timestamps")
-@click.option("--limit", default=1, metavar='NUM', help="Number of sets of passing nightlies to print")
+
+@cli.command("get-nightlies", short_help="Determine set(s) of accepted nightlies with matching contents for all architectures")
+@click.option("--matching", metavar="NIGHTLY_NAME", multiple=True, help="Only report nightlies with the same content as named nightly")
```

Comment:
I think the majority usecase for this would be using it with a passing x64 nightly.. any reason we're accepting multiple?

Reply:
At first I thought this was accepting nightlies of different arches, `--matching x64_nightly --matching p8_nightly` , but this seems to be intended for `--matching x64_nightly --matching another_x64_nightly`. But I'm not sure if we're restricting the first usecase? A bit confusing

Reply:
:+1: Yeah I don't mind having it in, it's pretty nifty!